examples/all-clusters-app/all-clusters-common/gen/attribute-size.cpp

```diff
@@ -30,6 +30,56 @@ using namespace chip;
 // and data is undefined.
 constexpr uint16_t kSizeLengthInBytes = 2u;
 
+// The first 2 bytes of a OCTET_STRING or a CHAR_STRING field in a list specifies the length.
+constexpr uint16_t kByteSpanInListLengthSize = 2;
+
+uint16_t GetByteSpanSize(const uint8_t * src)
+{
+    uint16_t size = *src & 0xFF;
+    size += (*(src + 1) << 8);
+    return size;
+}
+
+void ReadByteSpan(uint8_t * src, chip::ByteSpan * span)
+{
+    uint16_t entrySize = GetByteSpanSize(src);
+    *span              = chip::ByteSpan(src + kByteSpanInListLengthSize, entrySize);
+}
+
+void WriteByteSpan(uint8_t * dest, chip::ByteSpan * span)
+{
+    uint16_t entrySize = static_cast<uint16_t>(span->size());
+    dest[0]            = entrySize & 0xFF;
+    dest[1]            = (entrySize >> 8) & 0xFF;
+    memmove(dest + kByteSpanInListLengthSize, span->data(), entrySize);
+}
+
+uint16_t GetByteSpanOffset(uint8_t * buffer, uint16_t bufferLen, uint16_t index)
```

Comment:
This needs a comment on what it does - looking at name and arguments is not clear to me.